Modules/sre.h

```diff
@@ -71,18 +73,18 @@ typedef struct {
     Py_ssize_t pos, endpos;
     int isbytes;
     int charsize; /* character size */
+    /* current repeat context */
+    SRE_REPEAT *repeat;
     /* registers */
-    Py_ssize_t lastindex;
-    Py_ssize_t lastmark;
+    int32_t lastmark;
+    int32_t lastindex;
```

Comment:
Why? I you want just to save some memory, it is better to left it for other issue. Some local variables can change its type too.

Reply:
Yes, it can be changed it another issue.